apps/fluent-tester/src/RNTester/TestComponents/RadioGroup/RadioGroupTest.tsx

```diff
@@ -8,11 +10,13 @@ export const RadioGroupTest: React.FunctionComponent<{}> = () => {
   };
 
   return (
-    <RadioGroup label="This is a test RadioGroup" defaultSelectedKey="A" onChange={onChange}>
-      <RadioButton content="Option A" buttonKey="A" ariaLabel="Cool" />
-      <RadioButton content="Option B" buttonKey="B" />
-      <RadioButton content="Option C" buttonKey="C" disabled={true} />
-      <RadioButton content="Option D" buttonKey="D" />
-    </RadioGroup>
+    <View>
+      <RadioGroup label="This is a test RadioGroup" defaultSelectedKey="A" onChange={onChange}>
+        <RadioButton content="Option A" buttonKey="A" ariaLabel="Cool" testID={RADIOGROUP_TESTPAGE} />
```

Comment:
why is testID set on the first RadioButton here? Does it matter? Sometimes I see it on the Text components instead

Reply:
It doesn't matter. All those tests are checking is that something from the test page renders. In this case, there is no Text component so I chose the RadioButton